packages/frontend/web-ui/src/game/helpers/validateNumberOfPlayers.ts

```diff
@@ -0,0 +1,21 @@
+import { Either } from '../../common/models/Either';
+
+export const validateNumberOfPlayers = (
+  numberOfPlayers: number | undefined,
```

Comment:
I'd say `numberOfPlayers` should be a number
```suggestion
  numberOfPlayers: number,
```

Reply:
Done!